test/testcafe/framework/page-objects/EnrollEmailPageObject.js

```diff
@@ -21,12 +22,55 @@ export default class EnrollAuthenticatorPhonePageObject extends BasePageObject {
   }
 
   async enterCodeFromEmailLinkExists() {
-    const elCount = await Selector(ENTER_CODE_FROM_EMAIL).count;
-    return elCount === 1;
+    if (userVariables.v3) {
+      const elCount = await Selector('[type="button"]').count;
```

Comment:
This selector might be a little too brittle since it picks up all buttons on the page - Are you able to use something like
```ts
return this.form.getButton(ENTER_CODE_FROM_EMAIL_TEXT).exists;
```

Reply:
![Screen Shot 2023-01-03 at 9 55 54 AM](https://user-images.githubusercontent.com/27521424/210413863-5fdaa4db-dd07-4784-ba69-701002e5b153.png)
Turns out that `this.form.getButton` works for elements with `role=button`. But for this element, it only has `type=button` and there's no corresponding method. Is there any way we can add `role=button` for this element? @shawyu-okta

Reply:
`button` elements have an implied `role=button` so `getButton` should work. Let me know if that's not the case

Reply:
For some reason, this element doesn't have the implied `role=button`. `getButton` doesn't work and fails in this particular case.

Reply:
This doesn't work as expected. 
`this.form.getButton(ENTER_CODE_FROM_EMAIL_TEXT).exists;` fails in case the element doesn't exist. This is because the `getButton` method looks for an element with `role=button` and when it doesn't exist, it throws an error saying element not found.